src/queries/products.ts

```diff
@@ -9,9 +9,9 @@ export function useAvailableProducts() {
     "available-products",
     async () => {
       const res = await axios.get<AvailableProduct[]>(
-        `${API_PATHS.bff}/getProductsList`
+        `${API_PATHS.bff}/products`
       );
-      return res.data.message;
+      return res.data;
     }
   );
```

Comment:
These were my only changes for task-4, the diff is bigger because my reviewer hasn't checked my task-2 submission, thus the PR is still open